src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started its work");
```

Comment:
The message 'Bulldozer started its work' can be more informative. It should indicate the type of machine, which is 'Bulldozer', but also describe the work it is doing in more detail.